AutoCalculator/Scenes/Main/CurrencyConverter/CurrencyView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  CurrencyView.swift
+//  AutoCalculator
+//
+//  Created by 강성찬 on 2023-06-16.
+//
+
+import SwiftUI
+
+struct CurrencyView: View {
+
+    @StateObject var viewModel: CurrencyViewModel = CurrencyViewModel()
+    @FocusState private var inputIsFocused: Bool
+    
+    var body: some View {
```

Comment:
it would be nice to break down `body` into smaller pieces, such as making some private func or separate views

Reply:
yes